### PR TITLE
SHS-5582: Bug: Issue with Add/Select Media form field

### PR DESCRIPTION
**Pull request by mariannuar**
# READY FOR REVIEW

## Summary
Fix the "Sort by" label and form field that has incorrect styling when site editor uses the Media Library in CKEditor

## Need Review By (Date)
04/03

## Urgency
low

## Steps to Test
1. Create a Flexible Page on Colorful and Traditional
2. Select the Media Library in CKEditor and confirm the "Sort by" label and form field looks like this
3. Confirm any select/combox still looks as expected in other forms (Views for example)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss [hunk 1]
```diff
@@ -98,3 +98,16 @@ drupal-media {
 .ck-editor .ck.ck-toolbar.ck-toolbar_grouping > .ck-toolbar__items {
   flex-wrap: wrap;
 }
+
+body.gin--edit-form .views-exposed-form .select-preact {
```
Comment (cienvaras) [inline docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss]: Please move the code to `docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css`. That's the place to add admin theme styles.

File: docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss [hunk 2]
```diff
@@ -98,3 +98,16 @@ drupal-media {
...
+  label,
+  select {
+    visibility: visible !important;
+    height: auto !important;
+    position: relative !important;
+  }
+
+  div {
+    display: none;
+  }
```
Comment (cienvaras) [inline docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss]: This is not what we need. We still want to use the preact plugin (it has extra features —like search— that can be useful), but it should look like the regular select.

**Review by cienvaras — CHANGES_REQUESTED** [review]
@mariannuar Thanks! Some fixes needed though, please check the inline comments.

**Comment by mariannuar** [comment]
@cienvaras Thank you for your feedback! Now the preact plugin is looking similar to the regular select.

File: docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss [hunk 3]
```diff
@@ -97,4 +97,4 @@ drupal-media {
 // it causes overflow into the next column
 .ck-editor .ck.ck-toolbar.ck-toolbar_grouping > .ck-toolbar__items {
   flex-wrap: wrap;
-}
+}
```
Comment (cienvaras) [inline docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss]: Please remove this change.

File: docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css [hunk 4]
```diff
@@ -134,3 +134,33 @@ html:not(.gin--dark-mode) .user-logged-in[data-gin-accent="custom"] {
...
+
+body.gin--edit-form .views-exposed-form .select-preact div > div {
+    min-width: 0 !important;
+    max-width: 240px;
+  }
+  
+body.gin--edit-form .views-exposed-form .select-preact div > div > div:first-of-type {
+  font-size: 0.875rem !important;
+  font-weight: 525 !important;
+  color: #222330;
```
Comment (cienvaras) [inline docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css]: Gin has variables for most if its properties, please use them when available. In this case, `var(--gin-color-title)`.

**Review by cienvaras — CHANGES_REQUESTED** [review]
@mariannuar We're almost there, but there are still some inconsistencies that can be easily fixed:

- Label's line height should be 18px
- Button's vertical padding should be 7px, instead of 8px
- Border radius is smaller than other elements

Please use Gin CSS variables when available.

**Comment by mariannuar** [comment]
@cienvaras Thanks! Using gin variables now.

**Review by cienvaras — APPROVED** [review]
Looks great, thanks! 🚀

**Pull request merged**